src/js/main.js

```diff
@@ -46,6 +46,27 @@ function buildTimes() {
 function setSelectedTrain(destination) {
   selectedTrain = destination;
 }
+/*----------------------------------------------------------------------*\
+    Bartfunk Tunes
+\*----------------------------------------------------------------------*/
+function playSound(soundId) {
+  var soundFile = "not found";
+  switch (soundId) {
+    case 1:
+      soundFile = "high-ding.mp3";
+      break;
+    case 2:
+      soundFile = "low-ding.mp3";
+      break;
+    case 3:
+      soundFile = "woosh.mp3";
+      break;
+  }
+  if (soundFile == "not found") return;
+
+  var audio = new Audio("./audio/" + soundFile);
+  audio.play();
```

Comment:
This is a memory leak. 

Every time we will play a sound we will leak more and more memory, after the page is open for 15min you have . You do not need to create an audio element to play it, you need to play the element (which already exist) when you wish.